client/wfprev-war/src/main/angular/package.json

```diff
@@ -23,10 +23,19 @@
     "@angular/platform-browser": "18.2.3",
     "@angular/platform-browser-dynamic": "18.2.3",
     "@angular/router": "18.2.3",
+    "@busacca/ng-pick-datetime": "^13.0.1",
+    "@ngrx/store": "^18.1.1",
```

Comment:
Why are we adding this?

Reply:
We dont want to use anything ngrx

Reply:
I think it was in there from when I was pulling code out of wf1/core-ui. Removed now